### PR TITLE
Add group property to user

### PR DESCRIPTION
**Pull request by nishantrai2000**
This will add the ability for users to specify the group that they are in, and shows this under the user's properties.

### CODE REVIEW

File: attendance-app/lib/user.dart [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
   final String name;
   final String username;
+  final String group;
 
   static User fromMap(dynamic map) {
     return User(
       id: map['objectId'],
       name: map['name'],
       username: map['username'],
+      group: map['group']
```
Comment (jeffsieu) [inline attendance-app/lib/user.dart]: Add a trailing comma here

Comment (jeffsieu) [reply attendance-app/lib/user.dart]: But other than that, great work!

Comment (nishantrai2000) [reply attendance-app/lib/user.dart]: Thanks, Dad!

**Review by jeffsieu — APPROVED** [review]
@lordnishant Nicely done!

**Pull request merged**